webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DSRemoteTracingstoreClient.scala

```diff
@@ -0,0 +1,73 @@
+package com.scalableminds.webknossos.datastore.services
+
+import com.google.inject.Inject
+import com.scalableminds.util.tools.{Fox, FoxImplicits}
+import com.scalableminds.webknossos.datastore.DataStoreConfig
+import com.scalableminds.webknossos.datastore.dataformats.zarr.ZarrSegmentationLayer
+import com.scalableminds.webknossos.datastore.jzarr.{OmeNgffHeader, ZarrHeader}
+import com.scalableminds.webknossos.datastore.rpc.RPC
+import com.typesafe.scalalogging.LazyLogging
+import play.api.inject.ApplicationLifecycle
+import play.api.libs.json.JsObject
+
+class DSRemoteTracingstoreClient @Inject()(
+    rpc: RPC,
+    config: DataStoreConfig,
+    val lifecycle: ApplicationLifecycle,
+) extends LazyLogging
+    with FoxImplicits {
+
+  // TODO key check in tracing store
+  private val dataStoreKey: String = config.Datastore.key
+
+  def getZArray(tracingId: String, mag: String, tracingStoreUri: String, accessId: String): Fox[ZarrHeader] =
```

Comment:
```suggestion
  def getZArray(tracingId: String, mag: String, tracingStoreUri: String, accessToken: String): Fox[ZarrHeader] =
```
(lets’s try to use the same name everywhere)